components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/Oauth2ScopeConstants.java

```diff
@@ -59,7 +59,23 @@
         ERROR_CODE_FAILED_TO_GET_ALL_SCOPES_PAGINATION("51006",
                 "Error occurred while retrieving scopes with pagination."),
         ERROR_CODE_UNEXPECTED("51007", "Unexpected error"),
-        ERROR_CODE_FAILED_TO_GET_REQUESTED_SCOPES("51008", "Error occurred while retrieving requested scopes.");
+        ERROR_CODE_FAILED_TO_GET_REQUESTED_SCOPES("51008", "Error occurred while retrieving requested scopes."),
+        ERROR_CODE_FAILED_TO_RETRIEVE_USER_CONSENTS_FOR_APP("61001", "Error occurred while retrieving " +
```

Comment:
Why have we changed the error code range from 51xxx to 61xxx

Reply:
Fixed